### PR TITLE
[v0.3] add setup for denas and sda

### PR DESCRIPTION
**Pull request by zigzagcai**
add setup for e2eAIOK-denas and e2eAIOK-sda

### CODE REVIEW

**Comment by zigzagcai** [comment]
[PR#96](https://github.com/intel/e2eAIOK/pull/96) need to be merged firstly.

**Comment by zigzagcai** [comment]
Completed development for multi-package setup script

**Comment by zigzagcai** [comment]
1. Package All and Install
`python setup.py sdist`
2. Pakckage DeNas and Install
`python setup.py sdist --denas`
3. Pakcage SDA and Install
`python setup.py sdist --sda`

**Comment by zigzagcai** [comment]
- Install all of e2eAIOK
`python -m pip install e2eAIOK`
- Only install e2eAIOK DeNas:
`python -m pip install e2eAIOK-denas`
- Only Install e2eAIOK SDA:
`python -m pip install e2eAIOK-sda`

**Comment by Jian-Zhang** [comment]
@zigzagcai  @tianyil1  check the error msg.

**Comment by zigzagcai** [comment]
CICD passed. Please help to merge.

File: .github/workflows/e2eaiok_nightly_pypi.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build wheel
       run: |
         pip install build wheel
-        release_version=$(cat version | head -1)
+        release_version=$(cat e2eAIOK/version | head -1)
```
Comment (xuechendi) [inline .github/workflows/e2eaiok_nightly_pypi.yml:36]: why we are putting version in e2eAIOK? version is for all components

Comment (zigzagcai) [reply .github/workflows/e2eaiok_nightly_pypi.yml:36]: The original location of `version` file can not properly be ready when you run `python setup.py install`

Comment (zigzagcai) [reply .github/workflows/e2eaiok_nightly_pypi.yml:36]: `version` is for all components in e2eAIOK

**Review by xuechendi — APPROVED** [review]
LGTM

File: .github/workflows/e2eaiok_nightly_pypi.yml (hunk 1 above)
Comment (xuechendi) [reply .github/workflows/e2eaiok_nightly_pypi.yml:36]: I see, I'll merge this PR firstly and test with action then.

**Pull request merged**